app/assets/javascripts/spree/backend/solidus_i18n.js

```diff
@@ -1,2 +1,55 @@
 //= require spree/backend
 //= require_tree .
+
+Spree.routes.available_locales_api = Spree.pathFor('api/config/available_locales');
+
+var AvailableLocalesEdit = Backbone.View.extend({
+  initialize: function(){
+    var tbody = this.$("tbody[data-order-number][data-shipment-number]");
+
+  },
+
+  events: {
+    "click a.edit-available-locales": "toggleLocalesEdit",
+    "click a.cancel-available-locales": "toggleLocalesEdit",
+    "click a.save-available-locales": "saveLocales",
+  },
+
+  toggleLocalesEdit: function(e) {
+    e.preventDefault();
+    var store_id = this.$(e.currentTarget).data('store-id')
+
+    this.$("tr.store-id-"+store_id+".edit-available-locales").toggle();
+    this.$("tr.store-id-"+store_id+".show-available-locales").toggle();
+  },
+
+  saveLocales: function(e) {
+    e.preventDefault();
+    var _this = this
+    var store_id = this.$(e.currentTarget).data('store-id')
+    var preferred_available_locales = $('tr.store-id-'+store_id+' [name="store[preferred_available_locales][]"]')
+    $('#available-locales-presentation-text-'+store_id)
+
+    Spree.ajax({
+      type: "PUT",
+      url: Spree.routes.available_locales_api+'/'+store_id+'.json',
+      data: {
+        store: {
+          preferred_available_locales: preferred_available_locales.val(),
+        }
+      },
+      success: function(data){
+        _this.toggleLocalesEdit(e)
+        _this.$('#available-locales-presentation-text-'+store_id).text(
+          data.store.preferred_available_locales_presentation
+        )
+      }
+    });
+  }
+});
+
+$(function(){
```

Comment:
Please don't use jQuery DOM ready events. Use `Spree.ready`, so it is Turbolinks compatible.

And maybe we should put this in the view where we actually have the DOM elements available. Initializing directly inside the backbone file makes changes to JS initializing very hard.